react/QuantitySelector.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+* Quantity selector component.
+*/
+class QuantitySelector extends Component {
+  handleChange = (event) => {
+    const maxQuantity = this.props.maxQuantity
```

Comment:
Melhor seguir esse padrão:

const { maxQuantity, updateTotalPrice } = this.props

Reply:
Done!